docs/css/main.css

```diff
@@ -17,32 +17,62 @@ h1 {
   line-height: 48px;
   font-weight: normal;
 }
-
 h2 {
   font-size: 28px;
   line-height: 36px;
   font-weight: normal;
 }
-
 h3 {
   font-size: 20px;
   line-height: 30px;
   font-weight: normal;
 }
 
+h1.title {
+  max-width: 560px;
+  margin-bottom: 20px;
+}
+h1.title:after {
+  content: "";
+  background: #222;
+  height: 1px;
+  width: 32px;
+  display: block;
+  margin-top: 16px;
+}
+h1.title.center,
+h1.title.center:after,
+h2.center.description {
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+}
+h2.description {
+  margin-top: 20px;
+  margin-bottom: 24px;
+  max-width: 600px;
+  color: #999;
+}
+
+ol li,
+ul li {
+  padding-left: 12px;
+}
+
 article p {
   font-family: 'Roboto', 'Noto', sans-serif;
   font-size: 16px;
   line-height: 26px;
+  max-width: 720px;
```

Comment:
I think there is some inconsistent width introduced here, e.g. lists and stackblitz embeds are wider than the article.